lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -699,26 +699,30 @@ data TransactionInfo = TransactionInfo
 -- function where @x@ can be the transaction size in bytes or, a number of
 -- inputs + outputs.
 --
--- @a@ and @b@ are constant coefficients.
-data FeePolicy =
-    LinearFee (Quantity "lovelace" Double) (Quantity "lovelace/x" Double)
+-- @a@, @b@ and @c@ are constant coefficients.
+data FeePolicy = LinearFee
+    (Quantity "lovelace" Double)
+    (Quantity "lovelace/byte" Double)
+    (Quantity "lovelace/certificate" Double)
```

Comment:
:+1:

Reply:
(although, the second value is so-to-speak a `lovelace/number-of-input+output` :/ ... but it captures the same idea.